func/omp/CMakeLists.txt

```diff
@@ -1,7 +1,14 @@
 set(FAASM_USER omp)
 
-set(CMAKE_C_FLAGS "-fopenmp ${CMAKE_C_FLAGS}")
-set(CMAKE_CXX_FLAGS "-fopenmp ${CMAKE_CXX_FLAGS}")
+find_package(OpenMP QUIET)
```

Comment:
It's a little confusing as to whether this is used for the wasm version (i'm assuming it always fails?). It would be clearer to have:

```
if(FAASM_BUILD_TYPE STREQUAL "wasm")
    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fopenmp")
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")
else()
    find_package(OpenMP QUIET)
    include_directories(${OPENMP_INCLUDE_DIR})
    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
endif()
```